website/docs/guides/orchestration/running-dbt-cloud-jobs/2-fresh-rebuilds.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Running dbt Cloud jobs on fresh data"
+id: "running-dbt-cloud-jobs-on-fresh-data"
+slug: 2-fresh-rebuilds
+description: Learn how to avoid running models on stale data
+displayText: Fresh data rebuilds
+hoverSnippet: Learn how to avoid running models on stale data.
+---
+
+Many organizations encounter a situation in which their `sources` are updated on unpredictable intervals. In this case, it can be difficult to understand which downstream models actually need to run, and which can be left alone since their upstream sources haven't changed. 
+
+Fortunately, dbt Cloud supports using the source status selector to help!
+
+#### Configuring dbt Cloud jobs for source freshness
+
+To configure a dbt Cloud job with this strategy, include the `source freshness` option in the job configuration as shown in the screenshot below.
```

Comment:
Doesn't look like the screenshot is here yet. Also you should mention that you'll need to defer to an environment